go-apps/meep-gis-engine/api/swagger.yaml

```diff
@@ -283,6 +350,50 @@ definitions:
     example:
       active: true
       type: "MOBILITY"
+  DistanceParameters:
```

Comment:
Would it be clearer to call this object _TargetPoint_ and use _TargetRange_ for the withinRange endpoint?

Reply:
I changed it because i have no strong opinion.. i like the parameters better but as i said.. no strong opinion...